atom/browser/native_window_mac.mm

```diff
@@ -656,7 +656,8 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
       was_fullscreen_(false),
       zoom_to_page_width_(false),
       attention_request_id_(0),
-      title_bar_style_(NORMAL) {
+      title_bar_style_(NORMAL),
+      tabbing_identifier_(""){
```

Comment:
Space needed before `{`, perhaps we don't even need the `tabbing_identifier_` initializer here since it will be an empty string by default.